wp-content/themes/core/theme.json

```diff
@@ -1,10 +1,77 @@
 {
-	"$schema": "https://schemas.wp.org/trunk/theme.json",
-	"version": 2,
-	"settings": {
-		"layout": {
-			"contentSize": "80ch",
-			"wideSize": "1200px"
-		}
-	}
+  "$schema": "https://schemas.wp.org/trunk/theme.json",
+  "version": 2,
+  "settings": {
+    "color": {
+      "custom": false,
+      "customDuotone": false,
+      "customGradient": false,
+      "defaultGradients": false,
+      "defaultPalette": false,
+      "duotone": [],
+      "gradients": [],
+      "link": false,
```

Comment:
Note: I disabled link color as I think historically our design system has handled this and it's not likely to change